app/src/main/java/org/mozilla/focus/utils/CfrUtils.kt

```diff
@@ -54,22 +59,24 @@ class CfrUtils {
                         true
                     )
 
-                toolbarShieldIconCfrPopupWindow.showAsDropDown(
-                    shieldToolbarIcon, 0,
-                    TypedValue.applyDimension(
-                        TypedValue.COMPLEX_UNIT_DIP, SHIELD_ICON_CFR_Y_OFFSET,
-                        context.resources.displayMetrics
-                    ).roundToInt(),
-                    Gravity.BOTTOM
-                )
+                shieldToolbarIcon.post {
```

Comment:
Was the issue that the CFR popup was being displayed too early? Before it's anchor?
This might suggest that the flow for when the popup is shown can be better optimized but in the meantime this seems like a useful fix to stop a crash.